ui/app/routes/workspaces.ts

```diff
@@ -1,14 +1,21 @@
-import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 import ApiService from 'waypoint/services/api';
+import Route from '@ember/routing/route';
 import SessionService from 'ember-simple-auth/services/session';
+import { Workspace } from 'waypoint-pb';
+import { inject as service } from '@ember/service';
 
 export default class Workspaces extends Route {
   @service api!: ApiService;
   @service session!: SessionService;
 
   async redirect(): Promise<void> {
-    let workspaces = await this.api.listWorkspaces();
+    let workspaces: Workspace.AsObject[] = [];
+    try {
+      workspaces = await this.api.listWorkspaces();
+    } catch (error) {
+      // Send Authentication or other error if it exists
+      this.send('error', error);
```

Comment:
This makes me realize my mental model of how `redirect` behaves is incomplete with regard to errors.